.textlintrc.json

```
@@ -27,34 +27,87 @@
       "skip": [],
       "terms": [
         ".NET",
+        "Ajax",
         "Apache",
+        "API",
+        "AWS",
+        "Azure",
         "CNCF",
         "eBPF",
+        "ECMAScript",
+        "Elixir",
+        "Erlang",
+        "GCP",
+        "GitHub",
         "gRPC",
+        "GraphQL",
+        "ID",
+        "iOS",
         "HTTP",
         "Jaeger",
+        "Java",
+        "JavaScript",
+        "Jetty",
+        "JMX",
         "K8s",
+        "Kafka",
         "KubeCon",
         "Kubernetes",
+        "MacBook",
+        "macOS",
         "MySQL",
         "NGINX",
+        "npm",
```

NPM is a valid acronym, as we can gather from this Node.js page: https://nodejs.dev/en/learn/an-introduction-to-the-npm-package-manager/

```suggestion
        "NPM",
```

---

Oh, thanks! I removed it!